<refactor>: 컨벤션에 맞게 수정

kwonskdev opened the pull request:
일단 한거까지만 커밋해서 pr 날립니다. 수정할 거 있으면 말씀해주세요~

kwonskdev committed:
docs: autorec.py docstring 추가

- Encoder, Decoder, AutoRec class의 docstring 추가
- pre-commit-config 파일에서 super-init-not-called를 disable에 추가

kwonskdev committed:
refactor: 컨벤션에 맞게 수정

- 컨벤션에  수정
              no production code change
              no production code change

kwonskdev committed:
<refactor>: 컨벤션에 맞게 수정

- 컨벤션에  수정
              no production code change
              no production code change

otzslayer commented on deeprecs/utils/helper.py:
```diff
@@ -0,0 +1,26 @@
+from typing import Tuple
+
+
+def get_classes_from_module(module: ModuleType) -> Tuple[type, ...]:
```
재밌는 함수네요. 찬찬히 살펴보겠습니다 👀

kwonskdev replied:
> 

넵 수정하겠습니다

otzslayer commented:
이 형태로 쭉 작업하시면 되겠는데요? 큰 틀은 다 맞춰졌으니 자잘한건 다 Merge한 다음에 살펴봐도 괜찮을 것 같아요.
하나 걱정인건 저희끼리 한 번 다 합칠 때 굉장한 일이 벌어질 것 같다는거...? 🤪

otzslayer commented:
아 근데 커밋 타입에서 < > 빼주실 수 있나요?
그냥 refactor: ~~~ 로 해주시면 될 것 같아요.

kwonskdev commented:
@otzslayer
> 아 근데 커밋 타입에서 < > 빼주실 수 있나요?
그냥 refactor: ~~~ 로 해주시면 될 것 같아요.

넵 수정하겠습니다~

kwonskdev commented:
@otzslayer 
> 이 형태로 쭉 작업하시면 되겠는데요? 큰 틀은 다 맞춰졌으니 자잘한건 다 Merge한 다음에 살펴봐도 괜찮을 것 같아요.
하나 걱정인건 저희끼리 한 번 다 합칠 때 굉장한 일이 벌어질 것 같다는거...? 🤪

그게 같이 코드 짜는 묘미 아니겠습니까 기대됩니다

kwonskdev committed:
feat: AutoRec의 fit/predict 함수 작성

- AutoRec의 fit/predict 함수 작성
- _train_one_epoch 작성
- 생성자에 loss랑 optimizer 추가
- 기존에 부족했던 docstring 완성

kwonskdev committed:
data: MovieLens 100k 데이터 추가

- MovieLens 100k 데이터 (user_id, item_id) dataframe으로 변환
              no production code change
              no production code change

kwonskdev committed:
feat: 모델의 학습/추천에 사용할 데이터셋 클래스 작성

- 가장 기본적인 형태의 SampleDataset클래스만 작성
              no production code change
              no production code change

kwonskdev committed:
test: model test하는 pytest 함수 작성

- movielens 100k에 대해 autorec 결과 테스트하는 pytest 함수 작성
- test 결과 에러나는 autorec.py, data.py 오타 수정
              no production code change
              no production code change

kwonskdev committed:
merge: Remote autorec-base pull

- Merge branch 'autorec-base' of https://github.com/kwonskdev/deeprecs into autorec-base
